### PR TITLE
Webinar_GitHub_Branch - I am adding a message to display on my GitHub…

### PR DESCRIPTION
**Pull request by sharan2216**
@Sharan-LEAD11  Please review the code for Webinar GitHub Branch

### CODE REVIEW

**Comment by sharan2216** [comment]
File Change Path :------------
https://github.com/sharan2216/QAFox_GIT2/pull/1/files

**Comment by Sharan-Lead11** [comment]
@sharan2216  can u please  comment multiple lines

**Review by Sharan-Lead11 — COMMENTED** [review]
@sharan2216 can u please comment in multiple line again

File: tests/search.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 
+// new comments added here 1
```
Comment (Sharan-Lead11) [inline tests/search.py]: @sharan2216  please add multiple comment here

**Review by Sharan-Lead11 — COMMENTED** [review]
As i can see u have updated with correct code - Good to merge this branch

**Pull request merged**